client/styles/styles.scss

```diff
@@ -331,3 +331,17 @@ footer {
 .sk-three-bounce {
 	margin: 0;
 }
+
+.btn-share {
+	position: fixed;
+	right: 2%;
+	bottom: 2%;
+	visibility: visible;
```

Comment:
You can probably remove this line; visible is the default value for visibility afaik